.gitignore

```diff
@@ -1,7 +1,13 @@
 .ropeproject
-./zdpath
-./zoutput
+/zdpath
```

Comment:
/zdpath is absolution path?

Reply:
./zdpath does not work for me on the cluster. It does not ignore the directory then

Reply:
ok. I do not understand why the CI passed. I thought / means the root directory of a computer?

Reply:
But I think in the case of gitignore it means the root directory of the git project